lib/ui/widgets/dish_suggestion_card.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:food_app/ui/widgets/dish_suggestion_card_item.dart';
+
+class DishSuggestionCard extends StatefulWidget {
+  const DishSuggestionCard({super.key});
+
+  @override
+  State<DishSuggestionCard> createState() => _DishSuggestionCardState();
+}
+
+class _DishSuggestionCardState extends State<DishSuggestionCard> {
+  int selectedItem=0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      margin: const EdgeInsets.symmetric(vertical: 4),
+      child: Padding(
+        padding: const EdgeInsets.all(6.0),
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceBetween,
+          children: [
+            GestureDetector(
+                onTap: (){
+                  setState(() {
+                    selectedItem=0;
+                  });
+                },
+                child: DishSuggestionCardItem(dishIcon: Icons.local_pizza,dishName: "Pizza", isSelected: selectedItem==0?true:false,),),
+            GestureDetector(
+                onTap: (){
+                  setState(() {
+                    selectedItem=1;
+                  });
+                },
+                child: DishSuggestionCardItem(dishIcon: Icons.add,dishName: "Burger", isSelected: selectedItem==1?true:false,),),
```

Comment:
can we use the icon image from the UI Figma instead of `Icons.add`? If not, it's okay, we can change it later in the next PR or update